### PR TITLE
chore: add volunteer OkelloSam21

### PR DESCRIPTION
**Pull request by OkelloSam21**

### CODE REVIEW

**Comment by OkelloSam21** [comment]
Add OkelloSam files

File: data/users/OkelloSam21.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Okello Samuel",
+  "bio": "Java Developer",
+  "image": "https://https://avatars.githubusercontent.com/u/115361239?s",
+  "categories": [
+    {
+      "slug": "programming ",
```
Comment (codewithvoid) [inline data/users/OkelloSam21.json:7]: remove extra space in the end
```json
"slug": "programming"
```

File: data/users/OkelloSam21.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      "topics": ["Java,Javascript,kotlin"]
```
Comment (codewithvoid) [inline data/users/OkelloSam21.json:8]: Separate array elements for topics
```json
 "topics": ["Java", "Javascript", "Kotlin"]
```

File: data/users/OkelloSam21.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    }
+    
+   {
```
Comment (codewithvoid) [inline data/users/OkelloSam21.json:11]: lowercase slug and separate array elements here too

```json
   {
       "slug": "web-development",
       "topics":["HTML5", "CSS", "JavaScript"]         
    }
```

Comment (OkelloSam21) [reply data/users/OkelloSam21.json:11]: I've corrected this

**Review by OkelloSam21 — COMMENTED** [review]
done

**Review by codewithvoid — CHANGES_REQUESTED** [review]

File: data/users/OkelloSam21.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Okello Samuel",
+  "bio": "Java Developer",
+  "image": "https://https://avatars.githubusercontent.com/u/115361239?s",
```
[the remark this answers is not in the document]
Comment (OkelloSam21) [reply data/users/OkelloSam21.json]: I have fixed the url

File: data/users/OkelloSam21.json (hunk 3 above)
Comment (codewithvoid) [reply data/users/OkelloSam21.json:11]: Can you resolve this too?

File: data/users/OkelloSam21.json (hunk 1 above)
Comment (codewithvoid) [reply data/users/OkelloSam21.json:7]: This one is pending.

File: data/users/OkelloSam21.json (hunk 3 above)
Comment (OkelloSam21) [reply data/users/OkelloSam21.json:11]: Ive corrected the url and Ive removed javascript.

File: data/users/OkelloSam21.json (hunk 1 above)
Comment (OkelloSam21) [reply data/users/OkelloSam21.json]: done but let me check if my json file is valid firts

Comment (OkelloSam21) [reply data/users/OkelloSam21.json]: Done checking the file and its valid.

**Review by OkelloSam21 — COMMENTED** [review]
done

**Comment by OkelloSam21** [comment]
I think by now my json file is okay

**Review by codewithvoid — APPROVED** [review]
Thanks @OkelloSam21 for making the changes. Welcome!

**Pull request merged**